src/channel.js

```diff
@@ -693,7 +693,7 @@ export class Channel {
 				break;
 			case 'member.added':
 			case 'member.updated':
-				s.members = s.members.set(event.member.id, Immutable(event.member));
+				s.members = s.members.set(event.member.user_id, Immutable(event.member));
```

Comment:
There is no `id`!!

https://github.com/GetStream/chat/blob/5517979682a414290c6803b8559646a956d9b7d4/importer/member.go#L11